java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -12438,6 +12465,390 @@ public void sort_binary(BaseClient client) {
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void sort_with_pattern(BaseClient client) {
+        if (client instanceof GlideClusterClient) {
+            assumeTrue(SERVER_VERSION.isGreaterThanOrEqualTo("7.9.0"), "This feature added in version 8");
```

Comment:
Before merge, are you planning to change "7.9.0" to "8.0.0"?

Reply:
I have an issue to change that once valkey 8 is released